timescaledb/quick-start/ruby.md

```diff
@@ -4,727 +4,708 @@ excerpt: Get started with TimescaleDB for a Ruby application
 keywords: [Ruby]
 ---
 
-# Quick Start: Ruby and TimescaleDB
+import Intro from "versionContent/_partials/_quickstart-intro.mdx";
```

Comment:
Lana and I discovered that partial names are globally namespaced across all Markdown files, so we need to name them more specifically. The suggested convention is just CamelCasing the filename, so `QuickstartIntro` here.

Reply:
Thank you @charislam !!!!